packages/firebase_crashlytics/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.2.0
+
+* **Breaking**: Migration to the new Firebase Crashlytics SDK. Follow the [migration guide](https://firebase.google.com/docs/crashlytics/upgrade-sdk?platform=android) (steps 1 and 2) to update your app.
```

Comment:
Based on what I've seen, the team uses **Breaking change**

Reply:
@axel-op tagging you in case you missed this comment. Here's an example of where "Breaking change" has been used https://github.com/FirebaseExtended/flutterfire/blob/master/packages/firebase_auth/firebase_auth/CHANGELOG.md

Reply:
Done